src/ethereum/shanghai/fork.py

```diff
@@ -465,7 +470,14 @@ def apply_body(
     state : `ethereum.fork_types.State`
         State after all transactions have been executed.
     """
-    gas_available = block_gas_limit
+
+    # Calculate the additional gas added by the inclusion list and construct
+    # a map of addresses required.
+    inclusion_list_gas = sum(entry.gas_limit for entry in inclusion_list_summary)
+    inclusion_list_addresses = {entry.address: False for entry in inclusion_list_summary}
+
+    gas_available = block_gas_limit + inclusion_list_gas
```

Comment:
Is this right? I thought inclusion list would have its own gas limit

Reply:
it does! this is just adding the amount of gas used by the IL to the total gas limit of the block. so now if the IL has 2mm gas the total gas limit of the block should be 32mm.